reference/orders.v3.yml

```
@@ -274,7 +273,7 @@ paths:
     post:
       summary: Create a Refund
       description: |-
-        Creates a refund.
+        Creates a refund. When there are no payment method validation issues, the refund process is successful, and the refund payment request is scheduled. The refund payment request occurs asynchronously.
```

```suggestion
        Creates a refund. When there are no payment method validation issues, the refund process is successful and the refund payment request is scheduled. The payment request itself occurs asynchronously.
```
- list of two doesn't have a separating comma